drivers/display/display_st7796s.c

```diff
@@ -323,7 +323,7 @@ static const struct display_driver_api st7796s_api = {
 						SPI_OP_MODE_MASTER |		\
 						SPI_WORD_SET(8),		\
 						0),				\
-			.mode = MIPI_DBI_MODE_SPI_4WIRE,			\
+			.mode = DT_INST_PROP_OR(n, mipi_mode, 2),		\
```

Comment:
```suggestion
			.mode = DT_INST_PROP_OR(n, mipi_mode, MIPI_DBI_MODE_SPI_4WIRE),     \
```
We can still use the definitions here, they are included from the `mipi_dbi.h` header